src/views/secretTool.vue

```diff
@@ -74,7 +75,11 @@ export default {
 
       validatedInsights.forEach((insight_id, index) => {
         setTimeout(function() {
+          if(IS_DEVELOPMENT_MODE){
+          console.log(insight_id, 1)
+          } else {
           robotoffService.annotate(insight_id, 1);
+          }
```

Comment:
An easier way would be to modify `robotoffService.annotate`
Which you can find here
https://github.com/alexfauquette/openfoodfacts-hungergames-1/blob/master/src/robotoff.js#L6:L15

The structure is the following: in `robotoff.js` and `off.js` files you will the declaration of an object responsible of the interactions with off server (application server) and robotoff server (the one responsible of machine learning)

It's easier to cus post request in those files

For other post request, my bad, I'm not really clean coder on this project ^^

Reply:
Okay got it. I'll modify the robotoffService.annotate